README.md

```diff
@@ -88,3 +99,13 @@ Irene->Bob: countersigned updated state
 Alice->Irene: countersigned updated state
 end
 -->
+
+### Future work
+
+This repo originated as an entry for the ETHGlobal 2023 hackathon. Therefore some shortcuts were taken to enable an end-to-end demo given the time constraints. The following list identifies future areas for improvement if this foundation was used to build a production-ready application.
+
+1. Messaging: currently in-browser `BroadcastChannels` are used to communicate between the different users. A proper messaging service should allow users to exchange messages when they are on different machines with different IP addresses. Possible technologies include: WebSockets, WebRTC, HTTP requests, etc.
+2. Key management: currently the private signing keys used by each user to sign blockchain txs and state channel messages are stored in a static [.env](./.env) file. Since these keys control the movement of funds, they should be stored more securely, i.e. in Metamask or another crypto wallet.
+3. Intermediary funding: a key requirement for this state channel system to work is that Intermediaries need to ear-mark some of their own funds for end users so that off-chain payments can occur by rebalancing the funds designated for each user. This requires Intermediaries to essentially "lock up" some of their own money. Their should be incentives (e.g. payment processing fees) for the Intermediaries to provide this service. Eigenlayer re-staking could also be leveraged to unlock previously unused capital. Ethereum validator users who are already comfortable running reliable infrastructure are the type of actor who would likely be able to provide Intermediary services as well.
```

Comment:
```suggestion
3. Intermediary funding: a key requirement for this state channel system to work is that Intermediaries need to ear-mark some of their own funds for end users so that off-chain payments can occur by rebalancing the funds designated for each user. This requires Intermediaries to essentially "lock up" some of their own money. There should be incentives (e.g. payment processing fees) for the Intermediaries to provide this service. Eigenlayer re-staking could also be leveraged to unlock previously unused capital. Ethereum validator users who are already comfortable running reliable infrastructure are the type of actor who would likely be able to provide Intermediary services as well.
```

Reply:
Maybe add a link for eigenlayer?